### PR TITLE
Use date time in Hour instead of simple date

### PR DESCRIPTION
**Pull request by irismolinari**

### CODE REVIEW

File: app/models/report_entry.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@ def user
   end
 
   def date
-    I18n.l __getobj__.date
```
Comment (sigfrid) [inline app/models/report_entry.rb]: Time localization seems a better solution that string formatting.

File: db/migrate/20160912121407_change_date_format_in_hours.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class ChangeDateFormatInHours < ActiveRecord::Migration
+  def change
+  	change_column :hours, :date, :datetime
```
Comment (sigfrid) [inline db/migrate/20160912121407_change_date_format_in_hours.rb]: We should probably rename this column to `starting_time`.

File: spec/models/report_entry_spec.rb [hunk 3]
```diff
@@ -4,9 +4,9 @@
   subject(:report_entry) { ReportEntry.new(hour) }
   subject(:report_entry2) { ReportEntry.new(mileage) }
 
-  it "#date localized" do
-    expect(report_entry.date).to eq(I18n.l hour.date)
-    expect(report_entry2.date).to eq(I18n.l mileage.date)
+  it "#datetime" do
+    expect(report_entry.date).to eq(hour.date.to_s(:simple_datetime))
+    expect(report_entry2.date).to eq(mileage.date.to_s(:simple_datetime))
```
Comment (sigfrid) [inline spec/models/report_entry_spec.rb]: Do we still have `mileage` in the reports?

File: db/migrate/20160912121407_change_date_format_in_hours.rb (hunk 2 above)
Comment (irismolinari) [inline db/migrate/20160912121407_change_date_format_in_hours.rb]: Yes, I was planning to do this in the next updates.

File: spec/models/report_entry_spec.rb (hunk 3 above)
Comment (irismolinari) [inline spec/models/report_entry_spec.rb]: Reports can still manage both hour and mileage, even if we use them only with hours. As you see in line 5, a ReportEntry is initialized with mileage. If you prefer I can remove mileage from this spec.

File: app/models/report_entry.rb (hunk 1 above)
Comment (irismolinari) [inline app/models/report_entry.rb]: I thought using the same string formatting everywhere would be more consistent. If you prefer we could use localization in reports.

**Pull request merged**